pkg/humio/client.go

```diff
@@ -28,6 +28,7 @@ import (
 
 	"github.com/go-logr/logr"
 
+	"github.com/humio/cli/api"
```

Comment:
No need to introduce an import that is already there. Notice the line below, we can just reuse that import.
```suggestion
```